src/server/controllers/tasks.js

```diff
@@ -1,4 +1,5 @@
-import {Task} from '../models/tasks';
+import { Task } from "../models/tasks";
```

Comment:
I think you have some formatter that's going against our `eslint` ... please turn it off.
No spaces between the curly braces and quotes must be single, not double.